### ISSUE TITLE
Meta Performance per dollar should take into account playability

### ISSUE THREAD

**Issue by BryanQuigley** [phoronix-test-suite/phoronix-test-suite#77]
These meta results are awesome: https://www.phoronix.com/scan.php?page=article&item=nvidia-dollar-power&num=8.  

The only thing missing would be someway in the charts to note that the 750 Ti for instance can't actually play some of those titles well enough at 4K.  I'm not sure the best way to show this on the chart.

Just using 60 FPS as passing is OK, but having it line up to the refresh rate or be customizable by test might be useful to (Civ BE doesn't need 60 FPS to be playable).

**Comment by michaellarabel** [comment]
Hi, great to hear you like the new features and the feedback.

Yes, I thought about it but unfortunately haven't thought of a good way to handle it:

- Could introduce like a new "MinimumUsableValue" tag for test profiles to optionally say what result value is acceptable for a good experience, but that could still be somewhat differing based upon user's refresh rate and other factors. Then again this tag would really only cater to graphical/gaming tests mostly with there being no lower threshold on other test profiles like ray-tracing and media encoding.

- Tossing out results from the perf-per-dollar/Watt based on a certain threshold could skew the results if the user doesn't understand it's for games that are considered playable.

So long story short, yes it would be useful but as of right now haven't thought of a good way to be able to represent it. If you or anyone else has ideas feel free to chime in.

**Comment by L1ghtn1ng** [comment]
What about a tag that said can the graphics card play games at 4k and depending on the fps either would be a yes or no?

**Comment by michaellarabel** [comment]
@L1ghtn1ng sounds like the MinimumUsableValue tag mentioned above, which would still be somewhat subjective based on the gamer, their monitor, and then regardless a matter of how to display the meta performance: if simply dropping that data, then it would skew the data and potentially confuse the user. Probably side-by-side original and dropped results for  the "passing" tests would be best but then still a matter of if it would make sense, etc. Plus having to decide on values for each of the relevant test profiles as their minimum values.

**Comment by L1ghtn1ng** [comment]
It's a slight adaptation of the minimum usable value. And the values that you decide would be is the game playable let's say at 30fps but anything lower it cannot. Just spit balling  ideas

**Comment by sudden6** [comment]
You could just display the fps inside the the fps/dollar charts, e.g. 0.6fps/dollar@57fps

**Comment by michaellarabel** [comment]
@sudden6 that's perhaps one of the better ideas I've heard thus far... It would be I think easily doable in my thinking about it right now with the current code-paths. So basically displaying the FPS/dollar (or whatever per dollar value) and then next to that also displaying the overall average. Will do some playing around in the next few days if any better ideas don't arrive. Thanks.

**Comment by darkbasic** [comment]
I think it's a good idea.

**Comment by michaellarabel** [comment]
The recent Git snapshots indicate the overall average while showing the performance-per-dollar meta results. Thanks for the suggestion.
